### PR TITLE
[Code] [Bug Fixes] Snowball collision, master client change

### PR DESCRIPTION
**Pull request by KristianKurk**

### CODE REVIEW

File: Project Bunny/Assets/Scripts/Networking/ArenaManager.cs [hunk 1]
```diff
@@ -165,6 +167,19 @@ public override void OnMasterClientSwitched(Photon.Realtime.Player newMasterClie
         //Kick everyone from the room if the master client changed (too many bugs to deal with otherwise)
         PhotonNetwork.LeaveRoom();
     }
+    public override void OnLeftRoom()
+    {
+        ScoreManager scoreManager = FindObjectOfType<ScoreManager>();
+        PhotonTeamsManager teamsManager = FindObjectOfType<PhotonTeamsManager>();
+
+        if (scoreManager != null)
```
Comment (George540) [inline Project Bunny/Assets/Scripts/Networking/ArenaManager.cs:175]: Put these if statements for brackets

File: Project Bunny/Assets/Scripts/Networking/RoomManager.cs [hunk 2]
```diff
@@ -113,9 +113,14 @@ private void InitialiseUI()
         public override void OnLeftRoom()
         {
             ScoreManager scoreManager = FindObjectOfType<ScoreManager>();
+            PhotonTeamsManager teamsManager = FindObjectOfType<PhotonTeamsManager>();
+
             if (scoreManager != null)
                 GameObject.Destroy(scoreManager.gameObject);
 
+            if (teamsManager != null)
```
Comment (George540) [inline Project Bunny/Assets/Scripts/Networking/RoomManager.cs:121]: same here

**Review by George540 — APPROVED** [review]
all good, just some syntax changes

**Review by Cappic-01 — APPROVED** [review]
I think we can use photonView instead of fetching the Component directly. Other than that same comments as George.

**Pull request merged**